schema/beneficial-ownership-statements.json

```diff
@@ -57,49 +66,53 @@
       }
     }
   },
+  "required": [
+    "version",
+    "statementGroups"
+  ],
   "definitions": {
     "BeneficialOwnershipStatement": {
       "title": "Beneficial Ownership Statement",
-      "description": "ToDo",
+      "description": "A beneficial ownership statement is made up of an entity, an interested party (a reference to an entity, natural person, arrangement or trust), details of the interest and provenance information.",
       "type": "object",
       "properties": {
-        "id": {
+        "statementID": {
           "$ref": "#/definitions/ID"
         },
         "statementDate": {
           "$ref": "#/definitions/StatementDate"
         },
-        "entity": {
-          "title":"Entity",
-          "description":"This entity is the subject of a beneficial ownership relationship.",
+        "subject": {
+          "title":"Subject",
+          "description":"The subject of a beneficial ownership relationship.",
+          "properties": {
+            "subjectEntity": {
+                  "$ref": "#/definitions/EntityStatementReference"
+                }
+              },
           "oneOf": [
-            {
-              "$ref": "#/definitions/EntityStatementReference"
-            },
-            {
-              "$ref": "#/definitions/EntityStatement"
-            }
+            { "required": [ "subjectEntity" ] }
           ]
         },
         "interestedParty": {
           "title":"Interested party",
           "description":"The interested party has some level of ownership or control over the entity referenced in this beneficial ownership statement.",
+          "type": "object",
+          "properties": {
+            "interestedEntity": {
```

Comment:
we could just say 'entity', 'person' or 'unspecified' here, as the nature of the relationship is given above